### PR TITLE
SRVOCF-528 Update tekton tasks links to 1.29

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
Serverless 1.29+

Issue:
https://issues.redhat.com/browse/SRVOCF-528

Link to docs preview:

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by rh-max** [comment]
@jrangelramos Could you PTAL?

File: modules/serverless-functions-creating-on-cluster-builds.adoc [hunk 1]
```diff
@@ -24,14 +24,14 @@ You can use the Knative (`kn`) CLI to initiate a function project build and then
...
-$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.28.0/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.29.0/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml
 ----
 
 .. Create the `kn func` deploy Tekton task to be able to deploy the function in the pipeline:
 +
 [source,terminal]
 ----
-$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.28.0/pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml
+$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.29.0/pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml
```
Comment (jrangelramos) [inline modules/serverless-functions-creating-on-cluster-builds.adoc:34]: same here 
```suggestion
$ oc apply -f https://raw.githubusercontent.com/openshift-knative/kn-plugin-func/serverless-1.29.0/pkg/pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml
```

**Review by jrangelramos — COMMENTED** [review]
hello.. just need a minor update on the links.

**Comment by rh-max** [comment]
@jrangelramos Thanks Jefferson!

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**